src-electron/db/query-command.js

```diff
@@ -139,6 +139,12 @@ async function selectAllAvailableClusterCommandDetailsFromEndpointTypes(
       incoming: x.INCOMING,
       outgoing: x.OUTGOING,
       mfgCommandCount: x.MANUFACTURING_SPECIFIC_COMMAND_COUNT,
+      hasSpecificResponse: dbApi.toDbBool(x.RESPONSE_REF),
```

Comment:
I really wish this used our normal command mapping, so we don't keep getting skew like this....

Reply:
I was going for that and saw some mismatches which made me want to do the map cleanup later. I will log a bug a cleanup a lot of these surplus maps and make it all go to db-mapping